reference/apps.html.markerb

```diff
@@ -9,24 +9,8 @@ On the Fly Platform a Fly App is an abstraction for a group of virtual machines
 
 Every customer VM on the Fly Platform lives in a Fly App. Anycast IPs, certificates, Fly Volumes, and custom domains belong to apps as well.
 
-Fly Launch commands, `fly launch` and `fly deploy`, create and update an app's VMs as a unit with a single [configuration](/docs/reference/configuration/) and code base. Learn more about [Fly Launch](/docs/apps/).
+Fly Launch commands, `fly launch` and `fly deploy`, create and update an app's Machines as a unit with a single [configuration](/docs/reference/configuration/) and code base. Learn more about [Fly Launch](/docs/apps/).
 
 You can also create and configure [Machines](/docs/reference/machines/) independently of other Machines in the same app, using `fly machine run` or the Machines API.
 
-At the moment, there are two types of Fly App. 
-
-## Legacy (V1) Apps
-
-V1 Fly Apps are our original Firecracker apps. Instances of V1 Apps are orchestrated using Hashicorp Nomad. New organizations can no longer deploy V1 Apps to the Fly Platform, and existing V1 Apps are being migrated in phases. 
-
-## V2 Apps
-
-V2 Fly Apps are the new version of Fly Apps on the Fly Platform, running on [flyd](https://fly.io/blog/carving-the-scheduler-out-of-our-orchestrator/) instead of Nomad. The Firecracker VM building blocks of V2 Apps are [Machines](/docs/reference/machines/). 
-
-V2 Apps don't behave identically to V1 Apps. You have more control over where your VMs run; the flip side is that flyd won't magically move a Machine from one host to another the way Nomad does. There's a lot more information about this in [Carving the Scheduler Out of Our Orchestrator](/blog/carving-the-scheduler-out-of-our-orchestrator/).
-
-Machines within a single V2 App can also have different configurations and run different images, if you want. Machines [deployed by Fly Launch](/docs/apps/) are managed as a group, though, and run from the same image and the same config. 
-
-Your organization can have V1 Apps and V2 Apps on the same private network, running different layers of your full-stack application.
-
-V2 Apps features are evolving rapidly.
+Fly Apps run on [flyd](https://fly.io/blog/carving-the-scheduler-out-of-our-orchestrator/).
```

Comment:
This feels like a bit of a non sequitur to me. I don't mind if you like it though.

Reply:
Oh also, I can't reach it because it's old text, but I'd be tempted to change 
> On the Fly Platform a Fly App is an abstraction for a group of virtual machines running your code, along with the configuration, provisioned resources, and data we need to keep track of to run and route to your VMs.
>
> Every customer VM on the Fly Platform lives in a Fly App. Anycast IPs, certificates, Fly Volumes, and custom domains belong to apps as well.

to something more Machiney and shorter

Reply:
fixed it up a bit